lib/game_menu.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require_relative '../dependencies'
+
+class GameMenu
```

Comment:
What is the difference on GameMenu class and GameConsole class? It looks like they do the same thing.
Maybe you can combine them?

Reply:
it will be big class with many methods. i can try to do it, but is it worth to have file with many  methods?